internal/sponge/sponge.go

```diff
@@ -0,0 +1,120 @@
+// Package sponge provides support for item importing.
+package sponge
+
+import (
+	"reflect"
+
+	"github.com/ardanlabs/kit/db"
+	"github.com/ardanlabs/kit/log"
+	"github.com/ardanlabs/kit/web/app"
+	"github.com/cayleygraph/cayley"
+	"github.com/coralproject/shelf/internal/sponge/item"
+	"github.com/coralproject/shelf/internal/wire"
+)
+
+// Import imports an item into the items collections and into the graph database.
+func Import(context interface{}, db *db.DB, graph *cayley.Handle, itm *item.Item) error {
+	log.Dev(context, "Import", "Started : ID[%s]", itm.ID)
+
+	// See if the item already exists.
+	if itm.ID != "" {
+		items, err := item.GetByIDs(context, db, []string{itm.ID})
+		if err != nil {
+			if err != item.ErrNotFound {
+				log.Error(context, "Import", err, "Completed")
+				return err
+			}
+		}
+
+		// If we have an item already with this ID, we need to remove the
+		// existing relationships, assuming they are different.
+		if len(items) > 0 {
```

Comment:
@ardan-bkennedy We have an API for getting items by IDs.  We should probably have one that gets by ID singular.  That would solve this issue.  

Also, just FYI, we are checking for that "bug" in GetByIDS (i.e., if we get more items than expected).  Here we are just expecting either 1 or 0 items.